purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2432,17 +2433,21 @@ class PurchasesTest {
 
         mockStoreProduct(listOf(productIdSub), emptyList(), ProductType.SUBS)
 
+        val offeringIdentifier = "offering_a"
+        val purchaseOptionId = "purchase_option"
         capturedPurchasesUpdatedListener.captured.onPurchasesUpdated(
             getMockedPurchaseList(
                 productIdSub,
                 purchaseTokenSub,
                 ProductType.SUBS,
-                "offering_a"
+                offeringIdentifier,
+                purchaseOptionId = purchaseOptionId
```

Comment:
Nitpick and FFTI. It's a bit weird to just add this parameter as a named parameter... Should we change all of them instead?